headers/types/common/enums.h

```diff
@@ -3722,4 +3722,43 @@ enum wan_source_type {
 ENUM_8_BIT(wan_source_type);
 #pragma pack(pop)
 
+// Flags listed in the debug menu
+enum debug_flag {
+    DEBUG_FLAG_TEXT_SPEEDUP = 0,    // Increases text speed (x4)
+    DEBUG_FLAG_NO_SCREEN_FADE = 1,  // The screen doesn't fade to black when it's supposed to
+    DEBUG_FLAG_SOUND_OFF = 2,       // Disables all sounds?
+    DEBUG_FLAG_BGM_OFF = 3,         // Background music won't be played
+    DEBUG_FLAG_SE_OFF = 4,          // Sound effects won't be played
+    DEBUG_FLAG_STAGE_NPC_DUMMY = 5, // Unknown purpose
+    // Was supposed to throw an error if text overflowed the textbox it was contained in, but it
+    // does nothing in the final game.
+    DEBUG_FLAG_TEXT_LIMIT_CHECK = 6,
+    DEBUG_FLAG_NO_CHEAT_CHECK = 7,             // Unknown purpose
+    DEBUG_FLAG_NO_PLUNGE_CHECK = 8,            // Unknown purpose
+    DEBUG_FLAG_DUNGEON_INFINITE_COMEBACK = 10, // Unknown purpose
```

Comment:
This sounds like infinite revives within a dungeon? Idk how you verified the purpose of each debug flag though.

Reply:
Nope, I already tested it and it does not prevent you from fainting.